src/main/kotlin/tanvd/grazi/grammar/Typo.kt

```diff
@@ -92,4 +97,22 @@ data class Typo(val location: Location, val info: Info, val fixes: List<String>
             }
         }
     }
+
+    fun isAtStart(): Boolean {
```

Comment:
better move to location :)

Reply:
And add parameter `skipWhitespace = true`. In soma cases we may not want to skip whitespaces

Reply:
Done